scheduler/scheduler.go

```diff
@@ -166,16 +166,27 @@ func GetDistroQueueInfo(distroID string, tasks []task.Task, maxDurationThreshold
 		if !opts.IncludesDependencies || checkDependenciesMet(&task, depCache) {
 			task.ExpectedDuration = duration
 			distroExpectedDuration += duration
-		}
-
-		if duration > maxDurationThreshold {
-			if !opts.IncludesDependencies || checkDependenciesMet(&task, depCache) {
+			startTime := task.ScheduledTime
+			if task.DependenciesMetTime.After(startTime) {
+				startTime = task.DependenciesMetTime
+			}
+			task.WaitSinceDependenciesMet = time.Now().Sub(startTime)
```

Comment:
If dependencies haven't been met yet, then does it make sense for this field to be populated? since it implies that dependencies have now been met? Or is it more like "Wait For dependencies to be met"?

Reply:
If dependencies haven't been met, this field shouldn't be populated. I think I fixed this so that we won't try and populate this if `opts.IncludesDependencies` happens to be false.